### PR TITLE
Added test class.

### PR DESCRIPTION
**Pull request by jeongoks**
- **Added**
   - A `Class` for testing purposes and for the team to review and accept the change.

### CODE REVIEW

**Comment by LukasPedersen** [comment]
Hej

File: ITGuru.FourWheels/ITGuru.FourWheels.Data/Jasmin.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ITGuru.FourWheels.Data
+{
+    public class Jasmin
```
Comment (ColdSwordX) [inline ITGuru.FourWheels/ITGuru.FourWheels.Data/Jasmin.cs]: but why you?

Comment (jeongoks) [reply ITGuru.FourWheels/ITGuru.FourWheels.Data/Jasmin.cs:9]: Because I am Woman.

File: ITGuru.FourWheels/ITGuru.FourWheels.Data/Jasmin.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public int Age { get; set; }
```
Comment (ghost) [inline ITGuru.FourWheels/ITGuru.FourWheels.Data/Jasmin.cs]: Use uint instead?

Comment (jeongoks) [reply ITGuru.FourWheels/ITGuru.FourWheels.Data/Jasmin.cs]: I might ask stupidly but, why is that smarter than a proper int in this case?

File: ITGuru.FourWheels/ITGuru.FourWheels.Data/Jasmin.cs (hunk 1 above)
Comment (jeongoks) [reply ITGuru.FourWheels/ITGuru.FourWheels.Data/Jasmin.cs]: Changing this to a Person class

File: ITGuru.FourWheels/ITGuru.FourWheels.Data/Jasmin.cs (hunk 2 above)
Comment (ghost) [reply ITGuru.FourWheels/ITGuru.FourWheels.Data/Jasmin.cs]: I suppose that your age will never be negative. We can use the uint type to only allow positive numbers. Additionally, we can save space if we decide to use ushort instead (while still being albe to store the same positive numbers as an int can).

Comment (jeongoks) [reply ITGuru.FourWheels/ITGuru.FourWheels.Data/Jasmin.cs]: Got it. I will change that and make a new PR.

**Comment by jeongoks** [comment]
Changes made upon your requests.

File: ITGuru.FourWheels/ITGuru.FourWheels.Data/Jasmin.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+using System.Threading.Tasks;
+
+namespace ITGuru.FourWheels.Data
+{
+    public class Person
+    {
+        public uint Id { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public uint Age { get; set; }
```
Comment (ColdSwordX) [inline ITGuru.FourWheels/ITGuru.FourWheels.Data/Jasmin.cs]: Much better

**Review by ColdSwordX — APPROVED** [review]
now you cant have a negative age

File: ITGuru.FourWheels/ITGuru.FourWheels.Data/Jasmin.cs (hunk 3 above)
Comment (ghost) [reply ITGuru.FourWheels/ITGuru.FourWheels.Data/Jasmin.cs]: Да, идеально!

**Review by ghost — APPROVED** [review]

**Pull request merged**